modules/UI/UI.js

```diff
@@ -712,6 +712,14 @@ UI.setVideoMuted = function (id, muted) {
 };
 
 /**
+ * Triggers an update of remote video and large video displays so they may pick
+ * up any state changes that have occurred elsewhere.
+ *
+ * @returns {void}
+ */
+UI.updateAllVideos = () => VideoLayout.updateAllVideos();
```

Comment:
Just out of curiosity why not `UI.updateAllVideos = VideoLayout.updateAllVideos;`?